### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'split'

### ISSUE THREAD

**Issue by xoriole** [Tribler/tribler#6199]
Detected during #6198 [PR test](https://jenkins-ci.tribler.org/job/GH_Tribler_PR_Tests/job/PR_win64_pytest/1623/) on Windows 64 bit

```python
self = <test_triblertunnel_community.TestTriblerTunnelCommunity testMethod=test_perform_http_request_multipart>

    async def test_perform_http_request_multipart(self):
        """
        Test whether getting a large HTTP response works
        """
        self.add_node_to_experiment(self.create_node())
        self.nodes[1].overlay.settings.peer_flags.add(PEER_FLAG_EXIT_HTTP)
        await self.introduce_nodes()
    
        http_port = TestTriblerTunnelCommunity.get_free_port()
        http_tracker = HTTPTracker(http_port)
        http_tracker.tracker_info.add_info_about_infohash('0', 0, 0)
        http_tracker.tracker_info.infohashes['0']['downloaded'] = os.urandom(10000)
        await http_tracker.start()
        response = await self.nodes[0].overlay.perform_http_request(('127.0.0.1', http_tracker.port),
                                                                    b'GET /scrape?info_hash=0 HTTP/1.1\r\n\r\n')
    
>       self.assertEqual(response.split(b'\r\n')[0], b'HTTP/1.1 200 OK')
E       AttributeError: 'NoneType' object has no attribute 'split'

src\tribler-core\tribler_core\modules\tunnel\tests\test_triblertunnel_community.py:593: AttributeError
```
```

**Comment by drew2a** [comment]
Seems to be working now.

**Comment by drew2a** [comment]
Encountered it here: https://github.com/Tribler/tribler/pull/6708 
Failed job: https://jenkins-ci.tribler.org/job/GH_Tribler_PR_Tests/job/PR_win64_pytest/2900/

**Comment by drew2a** [comment]
All tested-related issues should be solved in #6851 after merging #6850 and #6855.

If the issue is encountered again, feel free to reopen it.
